example.jade

```diff
@@ -28,6 +28,7 @@ html(lang="en", class = ['classOne', 'classTwo'].join(','))
           span this one shouldn't higlight strings or... .other #things if else
           | this one shouldn't highlight strings, and the same goes for .keywords #ok ?
         div#paren.content.example(style = 'float: left;') Content .here #should be plain if for
+        +help("if this is .bolded #that's bad, we want a string","a second arg or else"); 
```

Comment:
This looks good on my local, but if you change `+help` to `+helpMe`, the uppercase "M" has no syntax highlighting. I think the regexp that is matching and highlighting the mixin lines is `jade-tag-re` (which intentionally doesn't highlight uppercase, although that could be its own discussion).

It looks like adding your new `jade-mixin-re` somewhere in the `font-lock-keywords` list might resolve that issue.